src/tables/cmap.js

```diff
@@ -361,12 +360,22 @@ function makeCmapTable(glyphs) {
     t.entrySelector = Math.log(t.searchRange / 2) / Math.log(2);
     t.rangeShift = t.segCountX2 - t.searchRange;
 
-    t.fields = t.fields.concat(endCounts);
+    for (let i = 0; i < endCounts.length; i++) {
```

Comment:
I was about to refactor this using a single `push()` call combined with the spread operator, but it turns out this is way more performant (see https://jsperf.app/cabimi). That's great!